### PR TITLE
Add explicit lowerings for bitwise_not

### PR DESCRIPTION
**Pull request by ezyang**
See https://github.com/pytorch/pytorch/issues/56273
For now I just want to work around the issue, but it's worth
investigating further why XLA doesn't just get these lowerings
for free.

Signed-off-by: Edward Z. Yang <ezyang@fb.com>

### CODE REVIEW

**Comment by ezyang** [comment]
cc @bdhirsh this will affect your codegen

**Comment by ezyang** [comment]
cc @JackCaoG

**Comment by ezyang** [comment]
I ran clang-format but apparently my version is wrong and it is still unhappy, can someone plz tell me how to get it or just run it on this PR for me, thx

**Comment by JackCaoG** [comment]
Thanks @ezyang ! Could you try `clang-format-7` ? https://github.com/pytorch/xla/blob/master/CONTRIBUTING.md#c-style-guide

File: scripts/gen.py [hunk 1]
```diff
@@ -116,6 +116,7 @@ class ArgTemplate(string.Template):
     'bernoulli_out': FuncOpts(),
     'binary_cross_entropy_out': FuncOpts(),
     'binary_cross_entropy_backward_out': FuncOpts(),
+    'bitwise_not': FuncOpts(),
```
Comment (JackCaoG) [inline scripts/gen.py:119]: I think you meant `bitwise_not_out` here?

**Review by JackCaoG — CHANGES_REQUESTED** [review]
Thanks @ezyang !

**Comment by ezyang** [comment]
This should get mooted by https://github.com/pytorch/pytorch/pull/56307

**Pull request closed without merging**